src/yang/ietf-qos-oper.yang

```diff
@@ -141,10 +141,10 @@ module ietf-qos-oper {
         "Bytes of packets dropped by meter";
     }
   }
-  grouping classifier-entry-statistics {
+  grouping classifier-statistics {
     description
-      "Statistics for a classifier entry";
-    leaf classifier-entry-name {
+      "Statistics for a classifier";
+    leaf classifier-name {
```

Comment:
Since this is under the list classifier, why even call it classifier-name? Call it just name.

Reply:
done.